app/controllers/hub/efile_errors_controller.rb

```diff
@@ -1,10 +1,11 @@
 module Hub
   class EfileErrorsController < Hub::BaseController
+    before_action :require_admin
     load_and_authorize_resource
     layout "hub"
 
     def index
-      @efile_errors = @efile_errors.order(:source, :code)
+      @efile_errors = EfileError.where(product_type: :ctc).order(:source, :code)
```

Comment:
for some reason because of changes in the abilities file it doesn't load anything into @efile_errors but @efile_error does; added require_admin at the top of here just for some added security could even do require state_file_admin if we want to be extra sure (this is a deprecated action anyways)